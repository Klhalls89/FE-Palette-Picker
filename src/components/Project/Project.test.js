import React from 'react';
import { shallow } from 'enzyme'
import Projects from './Project'

describe('Projects', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Projects/>)
  });
  it('Projects component should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });
})