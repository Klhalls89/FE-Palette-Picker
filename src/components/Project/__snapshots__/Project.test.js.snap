// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects Projects component should match the snapshot 1`] = `
<article
  className="project-area"
>
  <h2
    className="proj-name"
  >
    project name
  </h2>
  <div
    className="saved-palette-disp"
  >
    <h3
      className="pal-name"
    >
      palette name
    </h3>
    <i
      className="fas fa-trash-alt"
    />
  </div>
  <section
    className="saved-colors"
  >
    <div
      className="saved-color1"
    />
    <div
      className="saved-color2"
    />
    <div
      className="saved-color3"
    />
    <div
      className="saved-color4"
    />
    <div
      className="saved-color5"
    />
  </section>
</article>
`;
